### PR TITLE
Changed docs to use cli and some other changes

### PR DESCRIPTION
**Pull request by rohitsinha54**
- Docs uses cli commands now
- Removed app manager scripts and curl
- All commands has been tested to work

### CODE REVIEW

**Comment by rohitsinha54** [comment]
@abaranau @awholegunch : Please review.

File: MovieRecommender/README.rst [hunk 1]
```diff
@@ -43,19 +43,30 @@ Installation & Usage
 From the project root, build ``MovieRecommender`` with `Apache Maven <http://maven.apache.org/>`_ ::
 
   MAVEN_OPTS="-Xmx512m" mvn clean package
-  
+
+Note that the remaining commands assume that the cdap-cli.sh script is available on your PATH.
+If this is not the case, please add it::
+
+  export PATH=$PATH:<CDAP home>/bin
```
Comment (awholegunch) [inline MovieRecommender/README.rst]: What about this instead?

```
export PATH=$PATH:<cdap-home>/bin
```

File: MovieRecommender/README.rst [hunk 2]
```diff
@@ -43,19 +43,30 @@ Installation & Usage
...
+
+If you haven't already started a standalone CDAP installation, start it with the command::
+
+  cdap.sh start
+
 Deploy the Application to a CDAP instance defined by its host (defaults to localhost)::
 
-  bin/app-manager.sh --host [host] --action deploy
+  cdap-cli.sh deploy app target/MovieRecommender-0.2.0-SNAPSHOT.jar
```
Comment (awholegunch) [inline MovieRecommender/README.rst]: What about removing the version?

```
cdap-cli.sh deploy app target/MovieRecommender-<version>.jar
```

File: MovieRecommender/README.rst [hunk 3]
```diff
@@ -63,31 +74,33 @@ Ingest ``ratings`` and ``movies`` data::
...
 status (once done, it becomes STOPPED)::
 
-  bin/app-manager.sh --host [host] --action status
+  cdap-cli.sh get spark status MovieRecommender.RecommendationBuilder
   
-Once the Spark program is complete, you can query for recommendations via an HTTP request using the ``curl`` command::
-
-  curl -v \
-  -X GET 'http://localhost:10000/v2/apps/MovieRecommender/services/MovieRecommenderService/methods/recommend/1'
+Once the Spark program is complete, you can query ``MovieRecommenderService`` for recommendations::
```
Comment (awholegunch) [inline MovieRecommender/README.rst]: you can query the

**Comment by awholegunch** [comment]
A few comments, otherwise LGTM

**Pull request merged**